install_config/install/docker_registry.adoc

```diff
@@ -169,9 +169,41 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
-[[viewing-logs]]
 
+[[registry-ip-address]]
+=== Registry IP Address
+
+Because the integrated registry is referenced by its service IP address,
+if you decide to delete and recreate the docker-registry service,
+you should either take care to re-use its address,
+or reboot the cluster after creating the new docker-registry service.
+
+[[re-using-the-address]]
+==== Re-using the Address
```

Comment:
@tnguyen-rh  Did we ever talk about never going more than 3 ==='s down? There's a chance I made that rule up in my head...

I feel you could have the same effect here by calling "Re-using the Address" and "Rebooting the Cluster" into " *Option 1: Re-using the Address* " and the like.